Apsitvarkom.DataAccess/TidyingEventDatabaseRepository.cs

```diff
@@ -30,6 +30,12 @@ public async Task<IEnumerable<TidyingEvent>> GetAllAsync()
         return await _context.TidyingEvents.FirstOrDefaultAsync(propertyCondition);
     }
 
+    /// <inheritdoc />
+    public async Task<bool> ExistsByPropertyAsync(Expression<Func<TidyingEvent, bool>> propertyCondition)
+    {
+        return await _context.TidyingEvents.AnyAsync(propertyCondition);
+    }
```

Comment:
Same here

Reply:
same comment as above, if that seems alright